client/src/components/new-request/analyze-results.vue

```diff
@@ -735,27 +751,46 @@ export default class AnalyzeResults extends Vue {
 
 </script>
 
-<style scoped lang="sass">
-#name-search-bar
-  margin: -20px 0 20px 0
-  max-height: 40px
-  min-height: 40px
-.action
-  color: $error !important
-.approved
-  color: $approved !important
-.error-message
-  color: red
-  margin-left: 20px
-  margin-top: 15px
-.modal-activator
-  background-color: unset !important
-  color: $link !important
-  cursor: pointer !important
-  letter-spacing: unset !important
-  text-decoration: underline
-  text-transform: none !important
-.strike
-  text-decoration-line: line-through
+<style scoped lang="scss">
+@import '@/assets/scss/theme.scss';
 
+#analyze-results-container {
+  max-width: 1140px;
+}
+#name-search-bar {
+  margin: -20px 0 20px 0;
+  max-height: 40px;
+  min-height: 40px;
+}
+.action {
+  color: red !important;
+}
+.approved {
```

Comment:
How about moving "action" and "approved" to a base file for reuse?

Reply:
And maybe some of the classes below too! (eg, strike, maybe others)

Reply:
Oops, Vuetify already has a strike-through class.
https://v2.vuetifyjs.com/en/styles/text-and-typography/#text-decoration

Reply:
copy that, will do 👍

Reply:
Updated, left in Strike as it is being applied a little differently, not sure if the Vuetify class will slot in as a object option. Could work tho, but no luck so far.